common/tools/dev-tool/src/config/rollup.base.config.ts

```diff
@@ -103,11 +111,16 @@ function makeOnWarnForTesting(): (warning: RollupWarning, warn: WarningHandler)
 
 // #endregion
 
-export function makeBrowserTestConfig(): RollupOptions {
+export function makeBrowserTestConfig(pkg: PackageJson): RollupOptions {
+  // ./dist-esm/src/index.js -> ./dist-esm
+  // ./dist-esm/keyvault-keys/src/index.js -> ./dist-esm/keyvault-keys
+  const module = pkg["module"] ?? "dist-esm/src/index.js";
```

Comment:
There are only two packages that don't have `module`:

- core-asynciterator-polyfill
- mock-hub

The rest that don't have it are perf-tests and samples.

This default was really from a different time and I'd be fine with throwing an error in this case now assuming it doesn't break anything unexpected.

Reply:
The only way I can see this logic breaking is if the index module isn't in the same place relative to the test folder in any package. That is something that might be worth just spot-checking to make sure we don't break a random package with this change.